src/plugins/console/public/styles/_app.scss

```diff
@@ -109,15 +109,25 @@
   top: 0;
   // Adjust for possible scrollbars
   right: $euiSize;
-  // For IE11
-  min-width: $euiSizeXXL * 1.15;
   height: $euiSizeL;
   background-color: $euiFormBackgroundColor;
   border-radius: $euiSizeXS;
   box-shadow: 0 0 $euiSizeXS $euiSizeXS transparentize($euiShadowColor, .9);
   padding-top: $euiSize * .1;
 }
 
+.conApp__editorActions {
+  @extend .conApp__actionsPanel;
+  // For IE11
+  min-width: $euiSizeXXL * 1.15;
```

Comment:
Not part of your PR, but this is pretty funny :D 

You can ignore this, but it may as well be `$euiSizeXL * 2` which equates to 48 rather than this odd 46.

Reply:
For a panel that small, every pixel matters :D
Joking apart, 48 works fine so I updated it. Thanks for the suggestion!